mlflow/models/cli.py

```diff
@@ -155,6 +155,46 @@ def prepare_env(
     ).prepare_env(model_uri=model_uri)
 
 
+@commands.command("generate-dockerfile")
+@cli_args.MODEL_URI_BUILD_DOCKER
+@click.option(
+    "--output-directory",
+    "-d",
+    default="mlflow-dockerfile",
+    help="output directory for generating dockerfile",
+)
+@cli_args.ENV_MANAGER
+@cli_args.MLFLOW_HOME
+@cli_args.INSTALL_MLFLOW
+@cli_args.ENABLE_MLSERVER
+def generate_dockerfile(
+    model_uri, output_directory, env_manager, mlflow_home, install_mlflow, enable_mlserver
+):
+    """
+    Generates a directory with dockerfile whose default entrypoint serves an MLflow model at port
+    8080, using the python_function flavor. Function accepts a directory path and not just a
+    dockerfile path because if model is specified --- it will generate additional folder with
+    model files inside aforementioned directory This dockerfile defines an image that is
+    equivalent to the one produced by ``mlflow models build-docker``
+    """
+    env_manager = env_manager or _EnvManager.CONDA
+    backend = _get_flavor_backend(model_uri, docker_build=True, env_manager=env_manager)
+    if backend.can_build_image():
+        backend.generate_dockerfile(
+            model_uri,
+            output_directory,
+            mlflow_home=mlflow_home,
+            install_mlflow=install_mlflow,
+            enable_mlserver=enable_mlserver,
+        )
```

Comment:
After generation completes successfully, can we add a small `logger.info("Generated Dockerfile in directory <directory_path>")` message here?